__test__/lib/account-mock.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const faker = require('faker');
+const Account = require('../../model/account.js');
+
+const accountMock = module.exports = {};
+
+// Resolves -> request, account, token
+accountMock.create = (password) => {
```

Comment:
I like the idea of haveing the password be supplyied, that could make testing nice!

You could make it so there is a defaut password and then you only supply one when your testing for login